sandbox.html

```diff
@@ -15,13 +15,32 @@ <h1 v-if="message.length % 2 == 0">{{ message }}</h1>
         <button @click="list.push(list.length + 1)">Push Number</button>
         <button @click="list.pop()">Pop Number</button>
         <button @click="list.reverse()">Reverse List</button>
+        <button @click="show1 = !show1">Character List</button>
+        <button @click="show2 = !show2">Favourite list</button>
+        <button @click="favlist.push(prompt('Enter favourite character:'))">Add Favourite Character</button>
```

Comment:
`prompt` is not needed 😄

Reply:
Well directly it could be done - but generally we don't want to overcomplicate the function call.